pkg/development/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package development contains code that runs in the Playground.
+package development
```

Comment:
I think this is also used by `zed`. Maybe "used by other libraries to facilitate schema development?"